openlibrary/i18n/fr/messages.po

```diff
@@ -313,23 +316,23 @@ msgstr "Trier par"
 #: openlibrary/templates/work_search.html:114 view.html:141 view.html:143
 #: view.html:145 view.html:147
 msgid "Most Editions"
-msgstr "Le plus d'éditions"
+msgstr "Toutes les éditions"
```

Comment:
Most Editions != All Editions.

Reply:
Thank you for the feedback. I changed the translation back to the original translation.
I also changed the next two messages to singular form to keep it coherent with the translation of 'Most Editions' message as they are also part of sorting/ordering works option.

Reply:
Merci!